prime-router/src/test/kotlin/reportfilehistory/SubmissionHistoryTests.kt

```diff
@@ -14,7 +14,7 @@ import java.time.OffsetDateTime
 import java.util.UUID
 import kotlin.test.Test
 
-class SubmissionTests {
+class SubmissionHistoryTests {
```

Comment:
Please review jacoco test coverage, ensure major functions are covered by at least one test (at least happy path)

Reply:
Coverage is better now. jacoco is showing improved coverage but SonarCloud is still showing 0%